README.md

```diff
@@ -2,8 +2,8 @@ This is a basic gallery of the various AngularDart Components.
 
 * **[See the gallery](https://dart-lang.github.io/angular_components_example/)**
 * [See the code](https://github.com/dart-lang/angular_components_example/blob/master/lib/app_component.html)
-* [Use the package](https://pub.dartlang.org/packages/angular2_components)
-* [Browse the API](https://www.dartdocs.org/documentation/angular2_components/latest)
+* [Use the package](https://pub.dartlang.org/packages/angular_components)
+* [Browse the API](https://www.dartdocs.org/documentation/angular_components/latest)
 
 For a walk-through of how to *use* these components, see
 [Code Lab: AngularDart Components](https://webdev.dartlang.org/codelabs/angular2_components).
```

Comment:
We'll have to fix this once https://github.com/dart-lang/site-webdev/issues/563 is fixed